schema/internal/parser_checks.go

```diff
@@ -26,7 +26,7 @@ func CheckFileFormatField(fileFormat string, supportedFormatMajor, supportedForm
 	}
 
 	// Check that the major version number in the file is the same as what we expect.
-	if fileFormatParsed.Major() != uint64(supportedFormatMajor) {
+	if fileFormatParsed.Major() != uint64(supportedFormatMajor) { // nolint:gosec // Version can't be negative (overflow checked).
```

Comment:
Where is the overflow checked?

Reply:
We always control the value, so I assumed this was enough. I've added an explicit check.